tests/test_blinds.py

```diff
@@ -1,18 +1,29 @@
 """Test Blinds."""
 
+import json
+from typing import Any
+
 import pytest
 
 from pytradfri.device import Device
 
-from .devices import BLIND
+from tests.common import load_fixture
 
 
 @pytest.fixture(name="device")
-def device_fixture() -> Device:
-    """Return Device."""
-    return Device(BLIND)
+def device_fixture(request: pytest.FixtureRequest) -> Device:
```

Comment:
This fixture will be moved to a common module when all device types tests have been refactored.